docs: Update core getting started docs for clarity (#12146)
rook/rook: Documentation/Getting-Started/example-configurations.md

title: Example Configurations
---

> Configuration for Rook and Ceph can be configured in multiple ways to provide block devices, shared filesystem volumes or object storage in a kubernetes namespace. While several examples are provided to simplify storage setup, there are various settings that you will need to configure to optimize for your production environment.

Reviewer: ```suggestion
Configuration for Rook and Ceph can be configured in multiple ways to provide block devices, shared filesystem volumes or object storage in a kubernetes namespace. While several examples are provided to simplify storage setup, various settings will be needed to configure in order to optimize the production environment.
```
